src/Resources/Resources/Helper/RoleHelper.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
```

Comment:
Please add license header for new files. You can copy from other .cs files

Reply:
In fact, why not put it into a resx file (as a resource string?). Feel free to let me know if you don't know how